src/repositories/jobs.js

```diff
@@ -0,0 +1,85 @@
+const knex = requireKnex();
+const baseRepo = requireUtil("baseRepo");
+
+const create = async (payload) => {
+  try {
+    payload = baseRepo.addCreatedTimestamps(payload);
+    let result = await knex.transaction(async (trx) => {
+      const rows = await trx("jobs").insert(payload).returning("*");
+      return rows[0];
+    });
+    return result;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const getAllJobs = async (query) => {
+  try {
+    if (query && query.tags && query.tags.length > 0) {
+      var arr = query.tags.split(",");
+      return await getJobsByTags(arr);
```

Comment:
where is this function? what are tags? have we included them in our architecture? @mahesh-muttinti